src/Migrate/custom/Get-AzMigrateHCIJob.ps1

```diff
@@ -0,0 +1,176 @@
+
+# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+<#
+.Synopsis
+Retrieves the status of an Azure Migrate job.
+.Description
+The Get-AzMigrateHCIJob cmdlet retrives the status of an Azure Migrate job.
+.Link
+https://learn.microsoft.com/powershell/module/az.migrate/get-azmigratehcijob
+#>
+function Get-AzMigrateHCIJob {
+    [OutputType([Microsoft.Azure.PowerShell.Cmdlets.Migrate.Models.Api20210216Preview.IWorkflowModel])]
+    [CmdletBinding(DefaultParameterSetName = 'ListByName', PositionalBinding = $false)]
+    param(
+        [Parameter(ParameterSetName = 'GetById', Mandatory)]
+        [Microsoft.Azure.PowerShell.Cmdlets.Migrate.Category('Path')]
+        [System.String]
+        # Specifies the job id for which the details needs to be retrieved.
+        ${JobID},
```

Comment:
I think we can simplify JobID ->ID JobName -> Name because the subject of this cmdlet is Job